guide/webpack.config.js

```diff
@@ -32,6 +33,15 @@ module.exports = validate({
                 include: RegExp(__dirname + "/src"),
                 loaders:["react-hot", "babel-loader"]
             },
+	    {
+		test: /\.txt$/,
+		loader: 'raw-loader',
+		include: path.resolve(__dirname, 'src/examples/raw-code'),
+	    },
+	    {
+		test: /\.md$/,
+		loader: 'raw-loader',
+	    },
```

Comment:
To follow up on the import syntax. `raw` seems to just be shorthand for `raw-loader` both work the same. The difference however from the docs is that the bang in front `!raw-loader!` gives us pre transpiled code. Whereas without the leading bang `raw-loader!` from the docs example we get the transpiled version.

### With leading bang
<img width="1023" alt="screen shot 2017-07-10 at 10 44 50 am" src="https://user-images.githubusercontent.com/7366338/28031650-d29d295c-655c-11e7-944e-31f288254ba0.png">

### Without leading bang
<img width="986" alt="screen shot 2017-07-10 at 10 44 29 am" src="https://user-images.githubusercontent.com/7366338/28031655-d6b1c98a-655c-11e7-9bee-0a5891d4b94c.png">